exercises/concept/plane-tickets/.meta/exemplar.py

```diff
@@ -0,0 +1,44 @@
+""" Plane Tickets Exercise """
+
+def generate_seats(amount):
+    
```

Comment:
Are all these blank lines standard? Does the linter not complain?

Reply:
Actually, this is what the linter asked for. I tried to fix everything pylint asked.

Reply:
Sorry, I confused the test file with the exemplar. Will have another look.

Reply:
I fixed all these errors in a new commit